lib/core/security/tokenRepository.js

```diff
@@ -319,6 +329,19 @@ class TokenRepository extends Repository {
       .replace(TokenRepository.APIKEY_PREFIX, '');
   }
 
+  /** Verifies if a token is an ApiKey
+   * 
+   * @param {String} encodedToken
+   * @returns {Boolean}
+   */
+  isApiKey(token) {
```

Comment:
Instead of having a new method I would rather have an additional property in the `Token` model so anyone can easily know if a token is an authentication token or an API key